bliss/simulator/decoder.py

```diff
@@ -91,6 +92,22 @@ def source_renderers(self):
             SourceType.GALAXY: self.render_galaxy,
         }
 
+    def pixel_shift(self):
+        """Generate random pixel shift and corresponding WCS list to undo shifts."""
+        shift = np.random.uniform(-2, 2, (self.n_bands, 2))
+        wcs_base = WCS()
+        base = np.array([5.0, 5.0])
+        wcs_base.wcs.crpix = base
+        wcs = []
+        for i in range(self.n_bands):
+            if i == 2:
+                wcs.append(wcs_base.low_level_wcs)
+            else:
+                bnd_wcs = WCS()
+                bnd_wcs.wcs.crpix = base + shift[i]
```

Comment:
@zhixiangteoh This code might be useful for your work with coadds. (Just using WCS from DES may not work if you are simulating 80x80 images rather than large ones.) Sawan is dithering the images by a small amount here.